include/cadmium/engine/pdevs_dynamic_runner.hpp

```diff
@@ -62,8 +62,8 @@ namespace cadmium {
                  */
                 explicit runner(std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> coupled_model, const TIME &init_time)
                 : _top_coordinator(coupled_model) {
-                    LOGGER::template log<cadmium::logger::logger_global_time, TIME>(init_time);
-                    LOGGER::template log<cadmium::logger::logger_info, std::string>("Preparing model");
+                    LOGGER::template log<cadmium::logger::logger_global_time, cadmium::logger::run_global_time>(init_time);
+                    LOGGER::template log<cadmium::logger::logger_info, cadmium::logger::run_info>("Preparing model");
```

Comment:
Similarly to the comment about coor abreviation, I would keep runner full name here, I already got wondering if it refered to runner or to something about the "run" of the simulation.

Reply:
Ok, added to my todo list to be made in the next iteration